specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingSubscription.json

```diff
@@ -1,58 +1,81 @@
 {
   "swagger": "2.0",
   "info": {
-    "description": "Billing client provides access to billing resources for Azure subscriptions.",
     "title": "BillingManagementClient",
+    "description": "Documentation for Microsoft.Billing.",
+    "contact": {
+      "name": "Microsoft.Billing",
+      "url": "https://learn.microsoft.com/en-us/rest/api/billing/",
+      "email": "cabbpt@microsoft.com"
+    },
     "version": "2023-04-01"
   },
-  "host": "management.azure.com",
-  "schemes": [
-    "https"
-  ],
-  "consumes": [
-    "application/json"
-  ],
-  "produces": [
-    "application/json"
-  ],
   "paths": {
     "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/billingSubscriptions": {
```

Comment:
How impactful would it be for us to ensure we only have a single resource container per type of resource? I.e. only keep Microsoft.Billing/billingAccounts/billingSubscriptions, and remove Microsoft.Billing/billingAccounts/billingProfiles/billingSubscriptions, Microsoft.Billing/billingAccounts/billingProfiles/invoiceSections/billingSubscriptions, etc.?

Then if someone wants to get billing subscriptions by billing profile, they just query with a $filter.

Reply:
@ramkd12 @amberbhargava to comment here

Reply:
The resource id should only exist at one scope (billingAccounts/billingSubscriptions)...

But for PAS compatibility moving forward we need to expose list endpoints for each of the hierarchical scopes (billing account / billing profile / invoice section / customer)